### ISSUE TITLE
how to increase the training set data and also validation set data #2

### ISSUE THREAD

**Issue by diptnc** [amineHorseman/facial-expression-recognition-svm#10]
Hey there, I was trying your code can you please tell me how to increase the training set data and the validation set data.

![eqweqwe](https://user-images.githubusercontent.com/35225642/118140718-412a9a00-b426-11eb-9c34-d4b7297acbe8.JPG)
